### DISCUSSION TITLE
Request to join the team

### DISCUSSION THREAD

**Discussion by TasinAyon** [ROM-builders/temporary#23315] [General]
Hi Sir Apon77,

I would like to join your rom builders team please,

My build scripts,
https://github.com/TasinAyon/temporary/blob/main/build_rom.sh

Successfull Build: 
https://cirrus-ci.com/build/5354731241996288

Telegram ID: @TasinAyon

**Comment by Apon77** [comment]
do not use fake account, 
follow #learn instruction from telegram group
